src/util/mod.rs

```diff
@@ -45,6 +45,16 @@ pub mod timer;
 pub mod transport;
 pub mod worker;
 
+static PANIC_WHEN_KEY_EXCEED_BOUND: AtomicBool = AtomicBool::new(false);
+
+pub fn panic_when_key_exceed_bound() -> bool {
+    PANIC_WHEN_KEY_EXCEED_BOUND.load(Ordering::SeqCst)
```

Comment:
I think we can use Relaxed here.

Reply:
following PANIC_MARK can use Relaxed too

Reply:
I think it is ok, because it is very very rarely happen.